Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Formats/Format.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats
+{
+    public enum Format
```

Comment:
If you actually look at the formatter implementation, you will see that my code isn't like that. If a user wants to implement their own experience, they can either inherit the Formatter class, defining their own Default buttons, how to deal with Bold, etc. They can also insert additional custom buttons via XAML or code behind, remove default buttons via XAML or code behind.

The only purpose the Format enum has, is to Instantiate the RichText Formatter or the Markdown formatter. Set it to Custom, and then you can provide it your own inherited Formatter.

I have ensured that are plenty of places for the user to be able to adapt the code to suit their needs.

It is generic, yet you can still achieve similar functionality.